contract_show_invoice/__manifest__.py

```diff
@@ -1,20 +1,22 @@
 # -*- coding: utf-8 -*-
 # © 2015 Angel Moya <angel.moya@domatix.com>
 # © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# © 2017 Dave Burkholder <dave@thinkwelldesigns.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Contract Show Invoice',
     'summary': 'Button in contracts to show their invoices',
-    'version': '9.0.1.1.0',
+    'version': '10.0.1.0.0',
     'author': 'Domatix,'
               'Tecnativa,'
+              'Thinkwell Designs,'
```

Comment:
Yes, you're right. When I put that there, I wasn't aware of the README location. Found it in relation to another module, so I added it there as well. But now will yank it from the __manifest__ file.